### PR TITLE
Search update for collaboration

### PR DESCRIPTION
**Pull request by leogrim**
@2is10 @andrewconner @stkem 
1. Deploy Shoebox
2. Deploy Search
3. Wait for LibraryMembership reindexing
4. Deploy Search with new index version

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala [hunk 1]
```diff
@@ -153,10 +153,8 @@ object Shoebox extends Service {
     def canViewLibrary() = ServiceRoute(POST, "/internal/shoebox/libraries/canView")
     def newKeepsInLibraryForEmail(userId: Id[User], max: Int) = ServiceRoute(GET, "/internal/shoebox/database/newKeepsInLibraryForEmail", Param("userId", userId), Param("max", max))
     def getBasicKeeps(userId: Id[User]) = ServiceRoute(POST, "/internal/shoebox/database/getBasicKeeps", Param("userId", userId))
-    def getBasicLibraryStatistics() = ServiceRoute(POST, "/internal/shoebox/database/getBasicLibraryStatistics")
     def getBasicLibraryDetails() = ServiceRoute(POST, "/internal/shoebox/database/getBasicLibraryDetails")
     def getKeepCounts() = ServiceRoute(POST, "/internal/shoebox/database/getKeepCounts")
-    def getLibraryImageUrls() = ServiceRoute(POST, "/internal/shoebox/database/getLibraryImageUrls")
```
Comment (2is10) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala:159]: Just to be sure, these already have no live traffic hitting them?

File: kifi-backend/modules/search/app/com/keepit/search/controllers/website/WebsiteSearchController.scala [hunk 2]
```diff
@@ -35,6 +36,9 @@ object WebsiteSearchController {
   private[WebsiteSearchController] val maxKeepersShown = 20
   private[WebsiteSearchController] val maxLibrariesShown = 10
   private[WebsiteSearchController] val maxTagsShown = 15
+
+  private[WebsiteSearchController] val maxCollaboratorsShown = 3
+  private[WebsiteSearchController] val maxFollowersShown = 3
```
Comment (2is10) [inline kifi-backend/modules/search/app/com/keepit/search/controllers/website/WebsiteSearchController.scala]: I think we might show up to 5 collaborators, but this number is easy to tweak.

**Comment by 2is10** [comment]
Thanks.

File: kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala (hunk 1 above)
Comment (leogrim) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala:159]: Ah snap you're right, `getLibraryImageUrls` doesn't but `getBasicLibraryStatistics` is currently hit by the website, I'll figure something out.

**Pull request merged**